### DISCUSSION TITLE
Here's what's new in 1.1.3

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#139] [Announcements]
Not released yet, it's building right now.

Changes:
Mac M1 Builds
Bump tailwind version.

**Comment by imdevarsh** [comment]
is that it?

**Reply by imdevarsh** [reply]
oh btw bumping tailwind version changes nothing on the catalyst app unless its a minor release

**Reply by jdev082** [reply]
@webdev03 well considering the only one that can and knows how to add features kinda left...
yeah that's it

**Reply by imdevarsh** [reply]
What features do you need?

**Reply by jdev082** [reply]
at the moment, the webview is a limiting factor of these potential features:
password manager
download manager
there's no other way to do it with electron... it has to stop here unfortunately.

**Reply by jdev082** [reply]
it's either stop, or introduce a severe security vulnerability

**Reply by imdevarsh** [reply]
Well I have an idea. I'm playing on my computer right now but I think that if we give the list of passwords for that site on our list and then run the code. But that has a security problem - XSS could occur on the site and they have access to plaintext passwords

**Reply by imdevarsh** [reply]
I think basing the browser on top of Chromium or Firefox will be better

**Reply by jdev082** [reply]
no im not forking bloated chrome or FF.

**Reply by imdevarsh** [reply]
The browser is already including Chromium and NodeJS and all of Electron - that's why it takes up a lot of space on your computer

**Reply by jdev082** [reply]
I want my own browser not a fork of someone else's

**Reply by jdev082** [reply]
also go to the what's new in 1.1.6 post lmao
